pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1903,7 +1903,7 @@ assertNoAdditions {
     };
 
   markid = super.markid.overrideAttrs {
-    dependencies = [ self.nvim-treesitter ];
+    dependencies = [ self.nvim-treesitter-legacy ];
```

Comment:
I think the `configs` table is missing in main https://github.com/David-Kunz/markid/blob/9b249e4d2ad0767a1091017cd52264590e41a274/lua/markid.lua#L2-L3 so this might need update. Hasn't been updated in 2 years though